src/components/Proposal/VoteBox.tsx

```diff
@@ -169,12 +170,22 @@ class VoteBox extends React.Component<IContainerProps, IState> {
                 <span>{proposal.votesCount} Vote{proposal.votesCount === 1 ? "" : "s"}</span>
               </div>
               <div className={css.castVote}>
-                <button onClick={votingDisabled ? null : this.handleClickVote.bind(this, 1)} className={voteUpButtonClass}>
-                  <img src="/assets/images/Icon/vote/for-btn-selected.svg"/><span> For</span>
-                </button>
-                <button onClick={votingDisabled ? null : this.handleClickVote.bind(this, 2)} className={voteDownButtonClass}>
-                  <img src="/assets/images/Icon/vote/against-btn-selected.svg"/><span> Against</span>
-                </button>
+              {!votingDisabled ? 
+                <div>
+                  <button onClick={votingDisabled ? null : this.handleClickVote.bind(this, 1)} className={voteUpButtonClass}>
+                    <img src="/assets/images/Icon/vote/for-btn-selected-w.svg"/><span> For</span>
+                  </button>
+                  <button onClick={votingDisabled ? null : this.handleClickVote.bind(this, 2)} className={voteDownButtonClass}>
+                    <img src="/assets/images/Icon/vote/against-btn-selected-w.svg"/><span> Against</span>
+                  </button>
+                </div>
+              : 
+                <div className={css.votingDisabled}>
+                  <Tooltip overlay={<span>Voting has ended!</span>}>
```

Comment:
i thought we were going to customize the tooltip based on why voting is disabled. its not always because voting has ended

Reply:
OK Do we need copy from @A-Zak ?